ncsa-teststand/setup.env

```diff
@@ -1,8 +1,9 @@
-export CYCLE=c0020
+export CYCLE=c0021
 
 export SITE_PARTITION_PREFIX=ncsa
 
 export OSPL_MOUNT_POINT=/opt/OpenSpliceDDS/V6.10.4/HDE/x86_64.linux/etc/config/ospl.xml
```

Comment:
We should start using the `ospl.xml` that ships with the containers.

Reply:
OK. I'll write a note and do that.